### PR TITLE
Fizz buzz

### PR DESCRIPTION
**Pull request by yin-ka**
In this PR:
- We created a class called solver
- We created a method called factorial and implemented it
- We created a method called reverse and implemented it
- We created a method called fizzbuzz and implemented it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,115 @@
...
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+# 📖 TDD <a name="about-project"></a>
+> **TDD** This application is designed to help the librarian manage the library's collection of books and keep track of who borrows them. It provides an organized and efficient way to add students, teachers, and books, and maintain records of borrowings
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please add a proper description for your project because you used a description of an old project.

File: README.md [hunk 2]
```diff
@@ -1 +1,115 @@
...
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby - A Programmer's Best Friend</a></li>
+  </ul>
+</details>
+
+## Key Features <a name="key-features"></a>
+> - **Add new Book**
+> - **Create Teacher**
+> - **Create Student**
+> - **Save Records of who Borrowed Books**
```
Comment (omar-muhamad) [inline README.md]: - [ ] Kindly add key features of what you did inside this project, not for the old project.

File: README.md [hunk 3]
```diff
@@ -1 +1,115 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (omar-muhamad) [inline README.md]: - [ ] Kindly remove the live demo section because you don't have a deployment link yet.

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @yin-ka and @tajulafreen,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [ ] Kindly check the inline comments below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by Strangeal — APPROVED** [review]
Hi @yin-ka and @tajulafreen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Passing test ✅
- Pair Programming ✅
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Correct `.github/workflows` for linters ✅ 
- Well structured project ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**